horizons/ai/aiplayer/productionchain.py

```diff
@@ -234,7 +238,8 @@ def __init__(self, settlement_manager, resource_id, production_line, abstract_bu
 
 	def assign_identifier(self, prefix):
 		"""Recursively assign an identifier to this subtree to know which subtree owns which resource quota."""
-		self.identifier = '%s/%d,%d' % (prefix, self.resource_id, self.abstract_building.id)
+		self.identifier = '{}/{:d},{:d}'.index(
```

Comment:
index? Also, that line break is unnecessary. As well as some others in this pull request.